docs/applications.md

```diff
@@ -85,6 +85,64 @@ sharing the same configuration across observers, fluo provides a simple
 mechanism to set and access application specific configuration.  See the
 javadoc on [FluoClient].getAppConfiguration() for more details.
 
+Debugging Applications
+======================
+
+When monitoring [metrics][metrics] and a problem with an application is
```

Comment:
Could be
```
While monitoring [Fluo metrics][metrics] can detect problems (like too many transaction collisions) in a Fluo application, it can be difficult to debug the root cause of the problem.  To help debug Fluo applications, low-level logging of transactions can be turned on by setting the following loggers to TRACE:
``